docs/UserGuide.md

```diff
@@ -109,23 +109,115 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
-### Locating persons by name: `find`
+### Locating persons by name: `find person`
 
 Finds persons whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find person KEYWORD [MORE_KEYWORDS]`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
+* The search is case-insensitive. e.g. `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
 * Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
+* Only full words will be matched. e.g. `Han` will not match `Hans`
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+* `find person John` returns `john` and `John Doe`
+* `find person john betsy` returns `John Doe`, `Betsy Crowe`<br>
+  ![result for 'find john betsy'](images/findJohnBetsyResult.png)
+
+### Locating events by name: `find event`
+
+Finds events whose names contain any of the given keywords.
+
+Format: `find person KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g. `hiking` will match `Hiking`
+* The order of the keywords does not matter. e.g. `Hair Cut` will match `Cut Hair`
+* Only the name is searched.
+* Only full words will be matched. e.g. `Oscar` will not match `Oscars`
+* Persons matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `Hair Hiking` will return `Hair Cut`, `Park Hiking`, `Hiking`
+
+Examples:
+* `find event Hiking` returns `Hiking` and `Park Hiking`
+* `find event Hair Oscars` returns `Hair Cut`, `Oscars`<br>
+  ![result for 'find hair oscars'](images/findHairOscarsResult.png)
+
+### Viewing person by name: `view person`
+
+Views the details of a specific person whose name exactly matches the given keywords.
```

Comment:
i think we should emphasise that view is special in showing DETAILED details (address and email address), which makes it different from the rest. Maybe we could say: "Views additional detailed information of a specific person..... which includes Email address and address" sth like that maybe

Reply:
Good tips! We need to emphasize that! Fixed already.

Reply:
Thank you !